src/stores.js

```diff
@@ -4,253 +4,51 @@
 //
 // This file contains the model state for all the cards.
 
+const EosDiscoveryFeed = imports.gi.EosDiscoveryFeed;
 const Gio = imports.gi.Gio;
 const GObject = imports.gi.GObject;
 
 const Lang = imports.lang; 
 
-var CARD_STORE_TYPE_ARTICLE_CARD = 0;
-var CARD_STORE_TYPE_WORD_QUOTE_CARD = 1;
-var CARD_STORE_TYPE_ARTWORK_CARD = 2;
-var CARD_STORE_TYPE_AVAILABLE_APPS = 3;
-var CARD_STORE_TYPE_VIDEO_CARD = 4;
-var CARD_STORE_TYPE_NEWS_CARD = 5;
+var CARD_STORE_TYPE_UNSET = EosDiscoveryFeed.CardStoreType.UNSET;
+var CARD_STORE_TYPE_ARTICLE_CARD = EosDiscoveryFeed.CardStoreType.ARTICLE_CARD;
+var CARD_STORE_TYPE_WORD_QUOTE_CARD = EosDiscoveryFeed.CardStoreType.WORD_QUOTE_CARD;
+var CARD_STORE_TYPE_ARTWORK_CARD = EosDiscoveryFeed.CardStoreType.ARTWORK_CARD;
+var CARD_STORE_TYPE_AVAILABLE_APPS = EosDiscoveryFeed.CardStoreType.AVAILABLE_APPS;
+var CARD_STORE_TYPE_VIDEO_CARD = EosDiscoveryFeed.CardStoreType.VIDEO_CARD;
+var CARD_STORE_TYPE_NEWS_CARD = EosDiscoveryFeed.CardStoreType.NEWS_CARD;
```

Comment:
Can you change the rest of the code to use the values from `EosDiscoveryFeed.CardStoreType` directly instead?

Reply:
Hmm okay. I won't have time to do it this morning, but I'll try and backport my changes for this today.